backend/spec/spec_helper.rb

```diff
@@ -10,6 +10,19 @@
 require "webmock/rspec"
 WebMock.disable_net_connect!(allow_localhost: true)
 
+Geocoder.configure(lookup: :test, ip_lookup: :test)
+minneapolis_geo = {
+  # match the WeatherRetriever vcr stub
+  "coordinates" => [44.967486, -93.2897678],
+  "address" => "Minneapolis, Minnesota, USA",
+  "state" => "Minnesota",
+  "state_code" => "MN",
+  "country" => "United States",
+  "country_code" => "US",
+  "postal_code" => "55403"
+}
+Geocoder::Lookup::Test.set_default_stub([minneapolis_geo])
+
```

Comment:
I had to do this before any updates, spec failed consistently because position would not save/validate as its geocoder position was nil

```ruby
# backend/app/models/position.rb:21
  def geocoder_position
    @geocoder_position ||= Geocoder.search(postal_code).first
  end
```